DTAdmin/HTTPService.swift

```diff
@@ -0,0 +1,297 @@
+//
+//  HTTPService.swift
+//  DTAdmin
+//
+//  Created by Admin on 18.10.17.
+//  Copyright © 2017 if-ios-077. All rights reserved.
+//
+
+import Foundation
+
+class HTTPService {
+    //    static func login (userName:String, password:String){
+    static func login (completion: @escaping (HTTPURLResponse) -> ()){
+        let params = ["username": "admin", "password": "dtapi_admin"]
+        let url = URL(string: "http://vps9615.hyperhost.name/login/index")!
+        var request = URLRequest(url: url)
+        request.httpMethod = "POST"
+        request.setValue("application/json", forHTTPHeaderField: "Content-Type")
+        request.setValue("UTF-8", forHTTPHeaderField: "Charset")
+        let jsonData:Data
+        do {
+            jsonData = try JSONSerialization.data(withJSONObject: params, options: [])
+            request.httpBody = jsonData
+        } catch  {
+            print("some error")
```

Comment:
Very bad error handling. One unicorn died :)
Put a specific error message here instead. (e.g. "Could not serialize JSON" is better)